src/flint/fmpz_mat.jl

```diff
@@ -945,8 +952,8 @@ end
     lll_gram_with_transform(x::ZZMatrix, ctx::lll_ctx = lll_ctx(0.99, 0.51, :gram))
 
 Given the Gram matrix $x$ of a matrix $M$, compute a tuple $(L, T)$ where
-$L$ is the gram matrix of the LLL reduction of the matrix and $T$ is a
-transformation matrix so that $L = TM$.
+$L$ is the gram matrix of the LLL reduction of the matrix $M$ and $T$ is a
+transformation matrix so that $L = gram(TM)$, that is, $L = T^\top x T$.
```

Comment:
Is $T^\top x T$ correct or should it be $T x T^\top$? If $x$ is symmetric, it doesn't matter of course. Do we always assume that?

Reply:
Yes, I think `x` should be symmetric.